### ISSUE TITLE
Different Render Settings Override for each blend file in Loom Batch

### ISSUE THREAD

**Issue by DrSakuu** [p2or/blender-loom#50]
Right now you can override the render settings with a preset, but this is applied to all blend files in a batch. I render posters in Blender, and some of them are landscape and some are portrait resolution. If there was a button to set a preset for each blend file individually in the Loom Batch window, I could render all the posters in one go.

**Comment by p2or** [comment]
Hi @DrSakuu,

thanks for your feature request. To get an understanding of this: Why can't you just create a preset without the actual resolution? Or, for example, apply the preset, save a new version of that file and add this one to the queue? 

In general this should be easy to implement. The problem I personally see and the remaining question is: where to put the preset setting within each row without bloating the UI and destroying the current logic...? If you have a good idea or mockup to solve this, please let me know, I would appreciate your feedback.

Cheers,
Christian

**Comment by DrSakuu** [comment]
Yes, those are the same exact thoughts I had. I don't have a plan how to implement this nicely, but saving a separate blend file is a temporary solution. This can wait until you can make room in the interface.
